include/aws/testing/aws_test_harness.h

```diff
@@ -404,6 +404,9 @@ static inline int s_aws_run_test_case(struct aws_test_harness *harness) {
     AWS_ASSERT(harness->run);
 
 #if defined(_WIN32)
+#    if defined(AWS_WIN_ABORT_POP_UP_DISABLE)
+    _set_abort_behavior(0, _WRITE_ABORT_MSG | _CALL_REPORTFAULT);
+#    endif
```

Comment:
```suggestion
    _set_abort_behavior(0, _WRITE_ABORT_MSG | _CALL_REPORTFAULT);
```

I vote that we simply always do this.

Get rid of the AWS_WIN_ABORT_POP_UP_DISABLE cmake option, and cmakedefine, and --cmake-extra, and comment explaining why we need it in every ci.yml...

If we always set it, you'll still break in a debugger, if you have a debugger attached, see the implementation of [aws_fatal_assert()](https://github.com/awslabs/aws-c-common/blob/b0fe4ee08f754784cf39918e53f922bbd751941b/source/assert.c#L14), which calls `aws_debug_break()` which breaks into debugger if a debugger is attached, before calling abort().

Reply:
The only advantage of Windows' weird abort() popup is that it helps you break into a debugger. But no other platform does this, which is why we wrote our own logic to break into the debugger, which makes the popup totally useless for our tests...

always suppressing it will make tests on windows work like they do on all our other platforms

Reply:
Thanks, updated.